api/files/serializers.py

```diff
@@ -47,6 +47,14 @@ def to_internal_value(self, data):
         except StopIteration:
             self.fail('invalid_data')
 
+    def to_representation(self, value):
```

Comment:
I will change this but is this currently failing validation?  Before my changes, the href would still return null if no checkout url. Also, if a node doesn't have a parent, or there is no forked_from, the href is null.  Do these need to be changed as well?

![screen shot 2015-10-28 at 3 36 36 pm](https://cloud.githubusercontent.com/assets/9755598/10800710/aba897c6-7d89-11e5-9530-43ce919a1795.png)